mscore/CMakeLists.txt

```diff
@@ -302,13 +302,21 @@ else (NOT MSVC)
    set(_all_h_file "${PROJECT_SOURCE_DIR}/all.h")
 endif (NOT MSVC)
 
+file(GLOB MSCORE_PIANOROLL_SRC
+    ${CMAKE_CURRENT_LIST_DIR}/pianoroll/*.h
+    ${CMAKE_CURRENT_LIST_DIR}/pianoroll/*.cpp
+)
```

Comment:
See [this comment](https://github.com/musescore/MuseScore/pull/5931/files#r407495456) regarding use of `GLOB`.

Reply:
fixed